index.html

```diff
@@ -40,37 +119,60 @@
       /* color: #f59e0b; */
     }
 
+    /* #ffab00 */
+    /* #ffab00 */
     .textarea-box {
-      border-color: #f8a922;
+      border-color: darkslategrey;
+    }
+    .hidden .absolute .left-0 .font-light {
+      font-weight: 400;
+      font-size: 16px;
+    }
+    .font-light .text-amber-500 {
+      color: darkblue;
+      font-weight: 500;
     }
-
     textarea:focus,
     input:focus {
       outline: 0;
     }
-
+    span.developed {
+      font-size: large;
+      float: left;
+      font-weight: 500;
+    }
+    #spacebetweendeveloped {
+      padding-left: 10px;
+    }
     *:focus {
       outline: 0;
     }
+    /* div.border-b-2.w-screen.p-2.bg-amber-500.shadow-mg {
+      background-color: grey;
+      color: black;
+    } */
   </style>
 </head>
 
 <body>
   <div class="w-full h-screen flex-col justify-center relative">
     <!-- content-box -->
     <div class="flex flex-col">
-      <div class="border-b-2 w-screen p-2 bg-amber-500 shadow-mg">
-        <h1 class="text-4xl" id="Intro">
-          <a href="./index.html">Case Converter</a>
+      <!-- "border-b-2 w-screen p-2 bg-amber-500 shadow-mg" -->
```

Comment:
@vedamruta why commented out this line ?

Reply:
Actually, this was earlier used class name which I have modified but I have kept it for the maintainer to catch this change since he knows what he had earlier given as the class name.

Reply:
@chankruze it's not particularly a rainbow theme or so. I tried the rectangular colour theme in UI design. But then I found this salmon colour next to my rectangular theme colours that is why I added that. If there's any suggested changes anyone can raise an issue for that :)